src/templates/_includes/forms/checkboxGroup.twig

```diff
@@ -4,6 +4,7 @@
 
 {% set fieldsetAttributes = {
     class: ['checkbox-group'],
+    role: 'presentation',
```

Comment:
@brandonkelly I added this for both the `checkboxGroup` and the `radioGroup` as I wasn't sure if changing the `fieldset` element to a `div` would be problematic; if not let me know and I can modify the tag itself.

Reply:
Probably safe to just change the tag.

Reply:
Will do!